test/integration/bench/test_bench.py

```diff
@@ -0,0 +1,40 @@
+import os
+import signal
+import time
+from socket import AF_INET, SOCK_STREAM, socket
+from subprocess import PIPE, STDOUT, Popen
+from threading import Thread
+
+from utils import consume_lines, run_command_and_get_output
+
+
+def test_bench(cartridge_cmd, request, tmpdir):
+    base_cmd = [
+        cartridge_cmd, 'bench', '--duration=1'
+    ]
+    tarantool_cmd = ["tarantool",
+                     "-e", f"""box.cfg{{listen="127.0.0.1:3301",work_dir=[[{tmpdir}]]}}""",
+                     "-e", """box.schema.user.grant("guest","super",nil,nil,{if_not_exists=true})"""]
+
+    env = os.environ.copy()
+    process = Popen(tarantool_cmd, stdout=PIPE, stderr=STDOUT, env=env)
+    thread = Thread(target=consume_lines, args=["3301", process.stdout])
+    thread.start()
+
+    def kill():
+        process.send_signal(signal.SIGKILL)
+        if thread is not None:
+            thread.join(5)
+    request.addfinalizer(kill)
+
+    tntSocket = socket(AF_INET, SOCK_STREAM)
+    attempt = 10
+    while attempt > 0:
+        try:
+            time.sleep(1)
+            tntSocket.connect(('127.0.0.1', 3301))
+        except Exception:
```

Comment:
Don't you think the ``except`` with ``Exception`` is redundant here? I think the error class should be narrowed down

Reply:
The idea of ​​just closing the discussion without writing or correcting anything in code is bad.

Reply:
Sorry, misclick.
Code corrected with wait_for construction.